TASVideos/Pages/Shared/Components/GameName/Default.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿@model IList<GameNameModel>
 @if (Model.Any(g => g.IsSystem))
 {
-	<text><span>@Model.First().System</span></text>
+	<text><span>@Model.First().System games</span></text>
```

Comment:
Is this module also the one on system pages like https://tasvideos.org/GameResources/A7800 ? If so, I think this is a breaking change?

Reply:
On those it says "[system] games".